web_src/js/features/clipboard.js

```diff
@@ -1,21 +1,18 @@
 // For all DOM elements with [data-clipboard-target] or [data-clipboard-text], this copy-to-clipboard will work for them
 
-// TODO: replace these with toast-style notifications
 function onSuccess(btn) {
-  if (!btn.dataset.content) return;
   $(btn).popup('destroy');
   const oldContent = btn.dataset.content;
-  btn.dataset.content = btn.dataset.success;
+  btn.dataset.content = btn.dataset.success || '';
   $(btn).popup('show');
-  btn.dataset.content = oldContent;
+  btn.dataset.content = oldContent || '';
 }
 function onError(btn) {
-  if (!btn.dataset.content) return;
   const oldContent = btn.dataset.content;
   $(btn).popup('destroy');
-  btn.dataset.content = btn.dataset.error;
+  btn.dataset.content = btn.dataset.error || '';
   $(btn).popup('show');
-  btn.dataset.content = oldContent;
+  btn.dataset.content = oldContent || '';
```

Comment:
As above. 

Maybe `btn.dataset.content = btn.dataset.clipboardCopyError || i18n.common_clipboard_copy_error;`

and `btn.dataset.content = oldContent`

Reply:
Done (via global i18n vars)